iocore/net/UnixNet.cc

```diff
@@ -432,7 +432,7 @@ NetHandler::process_ready_list()
   }
 #else  /* !USE_EDGE_TRIGGER */
   while ((vc = read_ready_list.dequeue())) {
-    diags->set_override(vc->control.debug_override);
+    set_cont_flags(vc->control_flags);
```

Comment:
It would be best to conditionally set and unset this value.  Most of the time we will not have debugging on the vc.

Reply:
If it's not set, then it will remain whatever was the last VC's flags.

Reply:
set_cont_flags() always sets the ContFlag into thread_local.  You don't want to use the last VC's flag values, because the last VC's might have had debugging on.